charts/cp-schema-registry/templates/deployment.yaml

```diff
@@ -33,6 +33,11 @@ spec:
       {{- end }}
       {{- end }}
     spec:
+      securityContext:
+        runAsUser: 10001
```

Comment:
It can be nice to have a comment that indicate the name of the user which as this uid

Reply:
Agree but it's a no name user, there is no default user in the docker image itself, it was running as root.
So I picked up a random ID in the appropriate range